python/openai/docker/requirements_vllm.txt

```diff
@@ -0,0 +1,3 @@
+transformers==4.43.1
+# Llama3.1 vllm requirements
+vllm==0.5.3.post1
```

Comment:
should it be exactly this version or `>=` ?

Reply:
`==` for consistency during dev/testing, will ultimately use the deps shipped in the container it goes in